sysbuild/Kconfig.mcuboot

```diff
@@ -141,9 +141,9 @@ if BOOTLOADER_MCUBOOT
 
 choice BOOT_SIGNATURE_TYPE
 	default BOOT_SIGNATURE_TYPE_RSA if THINGY91_STATIC_PARTITIONS_FACTORY
+	default BOOT_SIGNATURE_TYPE_ED25519 if SOC_SERIES_NRF54LX
 	default BOOT_SIGNATURE_TYPE_ECDSA_P256 if ((SOC_NRF52840 || SOC_SERIES_NRF91X) && !BOARD_THINGY91_NRF9160 && !BOARD_THINGY91_NRF52840)
```

Comment:
not related to PR but noticed, that 5340 is not listed here i.e. uses RSA by default - is this correct?

Reply:
This was how it was in child/parent image and copied over, from what I remember it is inefficient and slow because it is done in software and was either wrongly added originally when it should have been added using ecdsa-p256 (i.e. years ago in child/parent) or the c132 in the bootloader does/did not support ecdsa-p256, and either way was not changed in the migration to sysbuild since it would break existing builds